cluster/images/canton/Dockerfile

```diff
@@ -32,23 +30,8 @@ COPY target/LICENSE.txt .
 
 RUN ln -s bin/canton splice-image-bin
 
-FROM fullstorydev/grpcurl:v1.9.2@sha256:0c3796edc8b6bb1814da2f11a554996237c0dd98380fce182efb1dc44c556ac1 AS grpcurl
-
-FROM eclipse-temurin:21-jdk-noble@sha256:5a65f334da5a91a66076735d78e3ae30483a2593ac108f830dcd59521f2535cd
-
-# Install platform-dependent dependencies at the last step to ensure correct paths and versions
-RUN apt-get update \
-   && DEBIAN_FRONTEND=noninteractive apt-get install -y libjemalloc2 \
-   && apt-get clean \
-   && rm -rf /var/lib/apt/lists/*
-
-COPY --from=build /usr/bin/tini /usr/bin/tini
-COPY --from=build /usr/bin/screen /usr/bin/screen
-COPY --from=grpcurl /bin/grpcurl /bin/grpcurl
-COPY --from=build /app/ /app/
-
-WORKDIR /app
-
+# TODO: Explain purpose of LABEL in a comment or remove it if not needed
```

Comment:
TODOs without an issue reference fail our CI. So you either need to remove the TODO or create an issue that you associate it with, I would suggest the former here. To answer the question in the TODO: These are standardized labels from https://github.com/opencontainers/image-spec/blob/main/annotations.md that are required by some image scanning tools.

Reply:
noted, thank you, I'll update it